charts/jackett/ci/ct-values.yaml

```diff
@@ -0,0 +1,10 @@
+jackett:
+  image:
+    organization: linuxserver
+    repository: jackett
+    tag: latest
```

Comment:
Should we use the same version as is in the `values.yaml` (`v0.16.1045-ls14`) here?  Floating to latest may have unintended consequences when testing an unrelated change to the chart and latest happens to have some breaking change.